src/helpers/shuffledImagesMemoBlocks.ts

```diff
@@ -0,0 +1,30 @@
+import { Image, Level } from "components/interfaces";
+
+export const shuffledImagesMemoBlocks = (
```

Comment:
Why is this function name a noun? If its usage corresponds a semantic purpose no problem. Nevertheless, it looks like it isn't.

Reply:
Moreover, It's a lambda function instead of a normal one, but you have the latter version in the majority of the functions of this module. So use either lambda or normal completely in order to get a standard.